Crash - Error: ENOENT: no such file or directory, open [...]watchy/src/../package.json

Watchy crashes on startup trying to resolve the location of its own package.json

<details open>
<summary>Versions</summary>

| Item | Version |
| --- | --- |
| node |18.16.0 (current LTS) |
| OS Name | Microsoft Windows 11 Pro |
| OS Version | 10.0.22621 N/A Build 22621 |
</details>

<details open>
<summary>Repro</summary

>
- Setup pnpm (if not already on the system)
> npm i -g pnpm

- Actual repro steps using pnpm
> mkdir repro-watchy-cli-16
> cd repro-watchy-cli-16
> pnpm init
> pnpm add -D watchy

- Run watchy to see crash
> pnpm watchy

</details>

<details open>
<summary>Logs</summary>

> ~\repro-watchy-cli-16 is 📦 v1.0.0 via  v18.16.0 took 2s
> ❯ pnpm watchy
> node:fs:601
>   handleErrorFromBinding(ctx);
>   ^
> 
> Error: ENOENT: no such file or directory, open '/C:/Users/Rocket/repro-watchy-cli-16/node_modules/.pnpm/watchy@0.10.0/node_modules/watchy/src/../package.json'
>     at Object.openSync (node:fs:601:3)
>     at Object.readFileSync (node:fs:469:35)
>     at file:///C:/Users/Rocket/repro-watchy-cli-16/node_modules/.pnpm/watchy@0.10.0/node_modules/watchy/src/cli.js:16:35
>     at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
>   errno: -4058,
>   syscall: 'open',
>   code: 'ENOENT',
>   path: '/C:/Users/Rocket/repro-watchy-cli-16/node_modules/.pnpm/watchy@0.10.0/node_modules/watchy/src/../package.json'
> }
> 
> Node.js v18.16.0
>  ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command failed with exit code 1: watchy
> 
> Command "watchy" not found.
</details>

--- a/src/cli.js
+++ b/src/cli.js
@@ -11,8 +11,7 @@ import _ from 'underscore';
 
 import watchy from './index.js';
 
-const { path: thisPath } = url.parse(import.meta.url);
-const packagePath = `${path.dirname(thisPath)}/../package.json`;
+const packagePath = new URL('../package.json', import.meta.url);
 const { version } = JSON.parse(fs.readFileSync(packagePath));
 
 const { env } = process;

